action_list_sample/action_list.js

```diff
@@ -39,11 +60,34 @@ const style = StyleSheet.create({
     container: {
         backgroundColor: "#ffffff"
     },
-
+    
```

Comment:
Don't include tabs on empty lines (there should be an option in your IDE of choice to automatically remove these).